src/components/DetailDrawer.tsx

```diff
@@ -195,6 +199,11 @@ export const DetailDrawer: React.FC<{
       <TableRowCustom>
         {columns.map(column => {
           if (shownColumns.includes(column.rowValueDescriptor)) {
+            const unformattedValue =
+              field[column.rowValueDescriptor as keyof ILookmlModelExploreField]
+            const displayValue = unformattedValue
+              ? String(unformattedValue)
+              : undefined
```

Comment:
`String(undefined)` returns `null` which is displayed to screen. Passing in `undefined` to the formatter creates a `<td />` with no children, as expected

Reply:
And `field[column.rowValueDescriptor as keyof ILookmlModelExploreField]` is undefined when a description doesn't exist, for example

Reply:
I would suggest destructuring `formatter` and renaming it with a capitalized letter. Then you can invoke it as a React component instead of as a function:

```typescript
columns.map(({ rowValueDescripter, maxWidth, minWidth, Formatter: formatter }) => 
  showColumns.includes(rowValueDescriptor) && (
    <TableDataCell
      color="text3"
      p="medium"
      pl="small"
      fontWeight="normal"
      key={rowValueDescriptor}
      maxWidth={maxWidth}
      minWidth={minWidth}
      onClick={
        column.rowValueDescriptor === "comment"
          ? commentsPane
          : detailsPane
      }
    >
      <Formatter
        x={field[rowValueDescriptor] ? String(field[rowValueDescriptor]) : undefined}
        isRow={true}
        field={field}
        commentCount={getFieldCommentsLenght(field.name)}
        canComment={canViewComments()}
        reader={permissions.reader}
    </TableDataCell>
  )
```